dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -64,7 +64,7 @@ decodeUrlPlusSign.name=Decode url plus sign
 decodeUrlPlusSign.desc=Decode the plus sign when it is part of the URL. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.decodeurlplussign. For servlet-5.0 and newer Servlet features, the default value is false.
 
 suppressHtmlRecursiveErrorOutput.name=Suppress recursive exception in the HTML response
-suppressHtmlRecursiveErrorOutput.desc=Suppresses the exception information from appearing in the HTML output when there is a recursive error that cannot be handled by an application's configured error page. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.suppressHtmlRecursiveErrorOutput.
+suppressHtmlRecursiveErrorOutput.desc=Suppresses the exception information from appearing in the HTML output when there is a recursive error that cannot be handled by an application's configured error page. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.suppressHtmlRecursiveErrorOutput. For servlet-5.0 and newer Servlet features, the default value is true and no longer configurable.
```

Comment:
change
the default value is true and no longer configurable.
to
the default value is true and is no longer configurable.